boa/src/builtins/object/mod.rs

```diff
@@ -824,6 +826,40 @@ impl Object {
             Ok(JsValue::new(false))
         }
     }
+
+    /// `Object.getOwnPropertyNames( object )`
+    ///
+    /// More information:
+    ///  - [ECMAScript reference][spec]
+    ///  - [MDN documentation][mdn]
+    ///
+    /// [spec]: https://tc39.es/ecma262/#sec-object.getownpropertynames
+    /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertyNames
+    pub fn get_own_property_names(
+        _: &JsValue,
+        args: &[JsValue],
+        context: &mut Context,
+    ) -> JsResult<JsValue> {
+        let o = args.get_or_undefined(0);
+        get_own_property_keys(o, GetOwnPropertyKeysType::String, context)
+    }
+
+    /// `Object.getOwnPropertySymbols( object )`
+    ///
+    /// More information:
+    ///  - [ECMAScript reference][spec]
+    ///  - [MDN documentation][mdn]
+    ///
+    /// [spec]: https://tc39.es/ecma262/#sec-object.getownpropertysymbols
+    /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertySymbols
+    pub fn get_own_property_symbols(
+        _: &JsValue,
+        args: &[JsValue],
+        context: &mut Context,
+    ) -> JsResult<JsValue> {
+        let o = args.get_or_undefined(0);
+        get_own_property_keys(o, GetOwnPropertyKeysType::Symbol, context)
+    }
```

Comment:
Just for completion sake, could you add the spec steps comments for these functions? Even if they're pretty much self-documenting, we would like to have the spec embedded on the code in case we need to optimize or refactor something :)

Reply:
Ah I see, sure!